examples/calibration/plot_calibration_curve.py

```diff
@@ -129,6 +129,6 @@ def plot_calibration_curve(est, name, fig_index):
 plot_calibration_curve(GaussianNB(), "Naive Bayes", 1)
 
 # Plot calibration curve for Linear SVC
-plot_calibration_curve(LinearSVC(), "SVC", 2)
+plot_calibration_curve(LinearSVC(max_iter=10000), "SVC", 2)
```

Comment:
Do we know how long this takes? It's a 100k samples run with 2 CV folds,  `10000` iterations each. Maybe increasing `tol` would be better here?

Reply:
tested the times, with and w/o max_iter here the times are virtually the same.